actions/matchActions.js

```diff
@@ -25,15 +25,10 @@ const getMatchError = (payload) => ({
   payload,
 });
 
-const matchApiCall = (matchId, host) => {
-  fetch(`${host}${url}${matchId}`, { credentials: 'include' })
-  .then(response => response.json())
-  .catch(error => error);
-};
-
-export const getMatch = (matchId, host = HOST_URL) => (dispatch) => {
+export const getMatch = (matchId) => (dispatch) => {
   dispatch(getMatchRequest());
-  matchApiCall(matchId, host)
+  return fetch(`${HOST_URL}${url}${matchId}`)
+    .then(response => response.json())
     .then(json => dispatch(getMatchOk(json)))
```

Comment:
Any reason for moving api call into action? Right now we are doing it in a lot of places, but in teh future it may be easier if we factor that code out into a separate modules for api calls so the action creators and thunks can be cleaner.